### ISSUE TITLE
Trained Models

### ISSUE THREAD

**Issue by AndreJFBico** [CS-Swansea/MC-NR-IQA#2]
Hi, im working with style transfer neural networks, i would like experiment with quality assessment of the resulting images.

If possible, can you share me the trained .h5 models?

**Comment by L2Program** [comment]
Hi Andre, you could try this set of weights which was used to train the fully convolutional model in the second experiment. This training run worked reasonably well and validation was performed on a validation set using vastly different images than the training set which posed quite a difficult NR-IQA task.

![ours - g perm-mb - o adam - j rot flip hsv - l charbonnier pcc - m ssim - p 64](https://user-images.githubusercontent.com/1933494/32344917-55dc8bde-c000-11e7-8ca9-6643ca4f5dfc.png)

[ours - g (perm-mb) - o (adam) - j (rot+flip+hsv) - l (charbonnier+pcc) - m (ssim) - p (64) - weights.zip](https://github.com/CS-Swansea/MC-NR-IQA/files/1438965/ours.-.g.perm-mb.-.o.adam.-.j.rot.flip.hsv.-.l.charbonnier.pcc.-.m.ssim.-.p.64.-.weights.zip)

However, it is worth mentioning that the key to this work was showing that No Reference, NR-IQA, methods should be trained directly on domain images containing the distribution of natural distortions that we want to evaluate during inference. 

This model was trained on a corpus of images rendered with Monte Carlo rendering algorithms such as Path Tracing, Bidirectional Path Tracing, Metropolis Light Transport, and others. Thus, it is trained to evaluate the quality of naturally occurring impulse noise distortions due to undersampling during the rendering process. This is done by training the model to minimize the loss of regressing a per pixel quality score compared to a Full Reference, FR-IQA, measure which in the case of the second experiment was SSIM. The per-pixel regression can then be globally averaged over the map to produce a scalar quality score.

It will not necessarily understand the distribution of distortions present in your images which will be due to very different reasons than the result of low-probability/high-energy samples in a Monte Carlo rendering process. 

Without a corpus of "ground-truth" style-transfered images for it to compare to using an FR-IQA measure like SSIM during training time it will be difficult to train it for the task of style-transfer image quality. 

That said, all the best of luck, I hope it works well. And please let me know of your results, success or not :)

**Comment by AndreJFBico** [comment]
Alright thanks ill give it a go, i appreciate you releasing the trained model!
